Cargo.toml

```diff
@@ -1,7 +1,20 @@
 [package]
 name = "license-exprs"
-version = "1.3.0"
+version = "1.4.0"
 authors = ["Without Boats <woboats@gmail.com>"]
-license = "Apache-2.0 OR MIT"
-description = "Validate SPDX 2.0 license expressions."
+license = "Apache-2.0/MIT"
```

Comment:
This is reverting #16; can you leave the `OR` form as it was?

Reply:
Fixed accordingly